packages/web-components/src/components/content-item-horizontal/content-item-horizontal.ts

```diff
@@ -26,12 +26,17 @@ class DDSContentItemHorizontal extends DDSContentItem {
   render() {
     return html`
       <div class="${prefix}--content-item-horizontal__row">
-        <div class="${prefix}--content-item-horizontal__col">
-          <slot name="eyebrow" @slotchange="${this._handleSlotChange}"></slot>
-          <slot name="heading"></slot>
+        <div class="${prefix}--content-item-horizontal__col ${prefix}--content-item-horizontal__col--1">
+          <div class="${prefix}--content-item-horizontal__heading-wrapper">
+            <slot name="eyebrow" @slotchange="${this._handleSlotChange}"></slot>
+            <slot name="heading"></slot>
+          </div>
+          <div class="${prefix}--content-item-horizontal__content-wrapper">
+            ${this._renderBody()}${this._renderFooter()}
+          </div>
         </div>
-        <div class="${prefix}--content-item-horizontal__col">
-          ${this._renderBody()}${this._renderFooter()}
+        <div class="${prefix}--content-item-horizontal__col ${prefix}--content-item-horizontal__col--2">
```

Comment:
Thanks for defining a slot for the thumbnail/image in the HTML return. I think we'll need to add a condition check (eg. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/card/card.ts#L130) within the result that adds in the `div` wrapper when the `thumbnail` or `image` slot is defined within the markup of the `<dds-content-item-horizontal>`. @annawen1 Let me know if you agree.

Also, @shixiedesign correct me if I'm wrong, but we'll need to account for the `<dds-video-player>` component that opens up the lightbox media viewer (https://ibmdotcom-web-components.mybluemix.net/?path=/story/components-video-player--with-lightbox-media-viewer)? if not, feel free to disregard my comment.

Reply:
> Thanks for defining a slot for the thumbnail/image in the HTML return. I think we'll need to add a condition check (eg. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/card/card.ts#L130) within the result that adds in the div wrapper when the thumbnail or image slot is defined within the markup of the <dds-content-item-horizontal>. @annawen1 Let me know if you agree.

yep, same for the `eyebrow` slot. We can check if the `thumbnail` attribute exists/is true to render the `thumbnail` and `eyebrow` slots.

```
{this.thumbnail ? <slot name="eyebrow" @slotchange="${this._handleSlotChange}"></slot> : undefined}
```

Reply:
@pavolprok @shixiedesign want to clarify again if we need to account for `video` as well or will it only be `image` for thumbnail?

Reply:
@annawen1 the thumbnails will only be images. No videos. ;-)

Reply:
@pavolprok Thanks for the confirmation!